client/landing/stepper/hooks/use-is-site-big-sky-eligible.ts

```diff
@@ -34,25 +33,15 @@ export function useIsBigSkyEligible( flowName?: string ) {
 
 	const isEligibleGoals = isGoalsBigSkyEligible( goals );
 	const isEligiblePlan = isPremiumPlan( product_slug ) || isBusinessPlan( product_slug );
-	const [ isLoadingBigsky, isBigSkyBeforePlansExperiment ] = useBigSkyBeforePlans();
-
-	if ( isLoadingBigsky ) {
-		return { isLoading: true, isEligible: null };
-	}
```

Comment:
Since `isLoadingBigsky` is always `false`, we can remove this check.

And since this was the only instance in which the hook was returning `isLoading: true`, we can also remove the `isLoading` prop from the hook's return value (since it would be always set to `false` anyway).